domain/model/bootstrap/bootstrap.go

```diff
@@ -148,10 +132,15 @@ func CreateReadOnlyModel(
 			CredentialOwner:   m.Credential.Owner,
 			CredentialName:    m.Credential.Name,
 			IsControllerModel: true,
+
+			// TODO (manadart 2024-01-13): Note that this comes from the arg.
```

Comment:
This is noted in the patch description. We shouldn't return a sparsely populated `core.Model` from the controller. This needs rethinking during the work by @tlm and @ycliuhw.

Reply:
Ack